media-libs/opencv/opencv-4.5.2-r3.ebuild

```diff
@@ -295,6 +295,9 @@ pkg_setup() {
 }
 
 src_prepare() {
+	# https://bugs.gentoo.org/838274
+	replace-flags -O3 -02
```

Comment:
It's `-O2`, as in a diatomic molecule of oxygen.

Reply:
Commit subject is wrong too.

Reply:
@mattst88 Good catch. 🤝